.github/scripts/Configure-AppxManifest.ps1

```diff
@@ -11,7 +11,17 @@ param(
     [string]$SecretGitHubOAuthClientId = ""
 )
 
+# Load Package.appxmanifest
 [xml]$xmlDoc = Get-Content $PackageManifestPath
+
+# Add namespaces
+$nsmgr = New-Object System.Xml.XmlNamespaceManager($xmlDoc.NameTable)
+$nsmgr.AddNamespace("pkg", "http://schemas.microsoft.com/appx/manifest/foundation/windows10")
+$nsmgr.AddNamespace("rescap", "http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities")
+$nsmgr.AddNamespace("uap", "http://schemas.microsoft.com/appx/manifest/uap/windows10")
+$nsmgr.AddNamespace("uap5", "http://schemas.microsoft.com/appx/manifest/uap/windows10/5")
```

Comment:
What are these for?

Reply:
You have to add namespaces to select nodes.